deluge/ui/gtkui/aboutdialog.py

```diff
@@ -7,29 +7,28 @@
 # See LICENSE for more details.
 #
 
-import gtk
-import pygtk
-
+import gi
+from gi.repository import GdkPixbuf, Gtk
 from deluge.common import get_pixmap, get_version, open_url_in_browser
 from deluge.ui.client import client
 from deluge.ui.gtkui.common import get_deluge_icon
 
-pygtk.require('2.0')
+gi.require_version('Gtk', '3.0')
 
 
 class AboutDialog:
     def __init__(self):
         def url_hook(dialog, url):
             open_url_in_browser(url)
-        gtk.about_dialog_set_url_hook(url_hook)
```

Comment:
Does removing this break clicking on the website links?

Also if you remove this line then also need to remove above `url_hook` function.